conf/ds-discover-behaviour.yaml

```diff
@@ -7,7 +7,8 @@
 #
 # This config is controlled by the individual developer and is not part of the code repository.
 #
-
+openapi:
```

Comment:
Please add some documentation as comments, stating what happens with the values under `openapi` and that they are expected to be overridden in the deploy-specific setups.